docs/source/pyopenms_in_r.rst

```diff
@@ -0,0 +1,243 @@
+
+You should now be able to interact with the OpenMS library and, for example, read and write mzML files:
+
+.. code-block:: R
+
+    library(reticulate)
+    ropenms=import("pyopenms", convert = FALSE)
+    exp = ropenms$MSExperiment()
+    ropenms$MzMLFile()$store("testfile.mzML", exp)
+
+which will create an empty mzML file called `testfile.mzML`.
+
+Getting help
+************
+
+Using the "reticulate" R package provides a way to access the pyopenms information 
+about the available functions and methods. We can inspect individual pyOpenMS objects 
+through the ``py_help`` function:
+
+.. code-block:: R
+
+    library(reticulate)
+    ropenms=import("pyopenms", convert = FALSE)
+    idXML=ropenms$IdXMLFile
+    py_help(idXML)
+
+    Help on class IdXMLFile in module pyopenms.pyopenms_4:
+
+   class IdXMLFile(__builtin__.object)
+    |  Methods defined here:
+    |  
+    |  __init__(...)
+    |      Cython signature: void IdXMLFile()
+    |  
+    |  load(...)
+    |      Cython signature: void load(String filename, libcpp_vector[ProteinIdentification] & protein_ids, libcpp_vector[PeptideIdentification] & peptide_ids)
+    [...]
+
+Alternatively, the autocompletion funcionality of RStudio can be used:
+
+.. code-block:: R
+
+    idXML=ropenms$IdXMLFile()
+    idXML$...load... (autocompletion)
+
+    (possibly screenshot)
+
+    Cython signature: void load(String filename, libcpp_vector[ProteinIdentification] & protein_ids, libcpp_vector[PeptideIdentification] & peptide_ids)
+
+In this case the idXML$load() function requires
+
+       - a filename as string
+       - an empty vector for pyopenms.ProteinIdentification objects
+       - an empty vector for pyopenms.PeptideIdentification objects
+
+Creating an empty R list() unfortunately is not equal to the empty python list []
+
+Therefore in this case we need to use the reticulate::r_to_py() function:
+
+.. code-block:: R
+
+    f="/OpenMS/OpenMS/share/OpenMS/examples/BSA/BSA1_OMSSA.idXML"
+    pepids=r_to_py(list())
+    protids=r_to_py(list())
+
+    idXML$load(f, pepids, protids)
+
+In order to get more information about the wrapped functions, we can also 
+consult the `pyOpenMS manual <http://proteomics.ethz.ch/pyOpenMS_Manual.pdf>`_ 
+which references to all wrapped functions.
+
+An example use case
+*******************
+
+Reading an mzML File
+^^^^^^^^^^^^^^^^^^^^
+
+pyOpenMS supports a variety of different files through the implementations in
+OpenMS. In order to read mass spectrometric data, we can download the `mzML
+example file <http://proteowizard.sourceforge.net/example_data/tiny.pwiz.1.1.mzML>`_
+
+.. code-block:: R
+
+    from urllib.request import urlretrieve
+    # from urllib import urlretrieve  # use this code for Python 2.x
+    library(reticulate)
+    ropenms=import("pyopenms", convert = FALSE)
+    mzML=ropenms$mzMLFile()
+    urlretrieve ("http://proteowizard.sourceforge.net/example_data/tiny.pwiz.1.1.mzML", "tiny.pwiz.1.1.mzML")
```

Comment:
this is Python code, that wont work... maybe `download.file("http://proteowizard.sourceforge.net/example_data/tiny.pwiz.1.1.mzML", "tiny.pwiz.1.1.mzML")` and replace this whole block with

```
    library(reticulate)
    ropenms=import("pyopenms", convert = FALSE)
    mzML=ropenms$MzMLFile()
    download.file("http://proteowizard.sourceforge.net/example_data/tiny.pwiz.1.1.mzML", "tiny.pwiz.1.1.mzML")
    exp = ropenms$MSExperiment()
    mzML$load("tiny.pwiz.1.1.mzML", exp)
```

Reply:
right, download.file works for me as well.

sorry I forgot about this one, when I copied it from the pyopenms tutorial.